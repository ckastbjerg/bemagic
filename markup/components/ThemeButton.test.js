/* eslint-env jest */
const ThemeButton = require('./ThemeButton');

test('returns a string of attributes from an object', () => {
    global.config = require('../../config');

    expect(ThemeButton({
        componentName: 'block',
        themeName: 'secondary',
        themeClasses: 'prefix-theme-secondary',
    })).toMatchSnapshot();
});
