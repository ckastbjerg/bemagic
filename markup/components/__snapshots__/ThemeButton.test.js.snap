// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns a string of attributes from an object 1`] = `
"
        <div
            class=\\"bemagic-theme-button bemagic-theme-button--small prefix-theme-secondary background js-page-theme\\"
            data-section-ref=\\"block\\"
            data-theme=\\"secondary\\"
        >
        </div>
    "
`;
